controllers/topics.controllers.js

```diff
@@ -31,7 +31,7 @@ exports.getTopics = (req, res, next) => {
 };
 
 exports.getArticles = (req, res, next) => {
-  selectArticles()
+  selectArticles(req.query.topic)
```

Comment:
It would be better practise to extract the topic query before passing it to the model. In the futre you may have other queries and this is going to get confusing/difficult to read.